src/Controller/Admin/LoginController.php

```diff
@@ -181,21 +187,14 @@ public function logoutAction(): void
      */
     public function loginCheckAction(Request $request, EventDispatcherInterface $dispatcher): RedirectResponse
     {
-        $event = new LoginRedirectEvent('pimcore_admin_login', [
-            'perspective' => strip_tags($request->get('perspective', '')),
-        ]);
-        $dispatcher->dispatch($event, AdminEvents::LOGIN_REDIRECT);
-
-        $url = $this->generateUrl($event->getRouteName(), $event->getRouteParams());
-
         // just in case the authenticator didn't redirect
-        return new RedirectResponse($url);
+        return new RedirectResponse($this->generateUrl('pimcore_admin_login'));
```

Comment:
```suggestion
        return new RedirectResponse($this->generateUrl('pimcore_admin_login', ['perspective' => strip_tags($request->get('perspective', ''))]));
```
this is needed, right? otherwise the perspective would be "lost"